api/CSSKeyframeRule.json

```diff
@@ -86,19 +151,19 @@
               "version_added": "12"
             },
             "opera_android": {
-              "version_added": true
+              "version_added": "12"
```

Comment:
Is this supported by test results, or just mirrored from Opera desktop? I suspect it's wrong, since the interface object isn't exposed.